dts/bindings/clock/nordic,nrf51-hfxo.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic nRF high-frequency crystal oscillator (nRF51 series)
+
+compatible: "nordic,nrf51-hfxo"
+
+include: fixed-clock.yaml
+
+properties:
+  clock-frequency:
+    const: 32000000
+
+  startup-time-us:
+    type: int
+    default: 1400
```

Comment:
Discussed offline. The bindings should not have default values as these are board specific. That will require a board designer to specify what is being used.

Decided to leave that for a followup